### PR TITLE
Remove split-pane widget

### LINKED ISSUE #1145: Remove Split Pane
Split pane is to be removed in favour of opinionated column layouts.

Column widgets issue: #1151

### PR DESCRIPTION
**Pull request by ericos**
**Type:** feature

The following has been addressed in the PR:

* [x] There is a related issue
* [ ] All code matches the [style guide](https://github.com/dojo/framework/blob/master/STYLE.md)
* [ ] Unit tests are included in the PR
* [ ] For new widgets, an entry has been added to the `.dojorc`
* [ ] Any widget variant uses `theme.compose` like [this](https://github.com/dojo/widgets/issues/847)
* [ ] WidgetProperties are exported

**Description:**
Removes the legacy `SplitPane` widget in anticipation of more opinionated column widgets.

Resolves #1145

### CODE REVIEW

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* Docs Deployment: https://dojowidgets-mndn1lqz6.now.sh
* Tests Deployment: https://dojowidgets-k2y4votro.now.sh

File: src/global-event/index copy.tsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
+import global from '@dojo/framework/shim/global';
```
Comment (rorticus) [inline src/global-event/index copy.tsx]: Don't think this file was intentional!

File: src/label/index copy.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
+import { WidgetBase } from '@dojo/framework/core/WidgetBase';
```
Comment (rorticus) [inline src/label/index copy.tsx]: Or this one!

File: src/global-event/index copy.tsx (hunk 1 above)
Comment (ericos) [reply src/global-event/index copy.tsx]: Yeah, not intentional. Thanks for catching this!

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* Docs Deployment: https://dojowidgets-47f076mur.now.sh
* Tests Deployment: https://dojowidgets-1czrcsivv.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* Docs Deployment: https://dojowidgets-723jk75fc.now.sh
* Tests Deployment: https://dojowidgets-rn6qy2t0d.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* Docs Deployment: https://dojowidgets-723jk75fc.now.sh
* Tests Deployment: https://dojowidgets-rn6qy2t0d.now.sh

**Pull request merged**